cmd/create/cluster/cmd.go

```diff
@@ -2001,6 +2016,58 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// If OCP version >= 4.10 only
+	var machinePoolRootDisk *ocm.Volume
+	// setting the machine pool root disk size is only supported on gp3 volumes
+	gp3SupportedVersion := "4.10.0"
+	if ocm.IsGreaterThanOrEqual(version, gp3SupportedVersion); err == nil {
+		var machinePoolRootDiskSizeStr string
+		if args.machinePoolRootDiskSize == "" {
+			// We don't need to parse the default since it's returned from the OCM API and AWS
+			// always defaults to GiB
+			machinePoolRootDiskSizeStr = gigybyteStringer(defaultMachinePoolRootDiskSize)
```

Comment:
If a user does not pass a size, do we even need to set this value. Can we not just handle it in the API as normal?

Reply:
We might still go through the interactive mode if it's not passed. I'll handle the case where the arg is empty and the interactive mode is disabled. Thanks.

Reply:
Cheers, personally I think it would be better. If the user does not pass anything, or selects default, we should do nothing. No need to pass the default to the API, lets let the backend decide the default.